src/hooks/__tests__/useScanApi.test.ts

```diff
@@ -295,18 +295,6 @@ describe('useGetScanJobsApi', () => {
     expect(mockOnAddAlert.mock.calls).toMatchSnapshot('getScanJobs, error');
   });
 
-  it('should process an API success response', () => {
-    const { callbackSuccess } = hookResult;
-
-    callbackSuccess({
-      data: {
-        results: [{ name: 'Lorem', id: '1' }]
-      }
-    });
-
-    expect(mockOnAddAlert.mock.calls).toMatchSnapshot('callbackSuccess');
-  });
-
```

Comment:
required: this test should be updated instead of removed to maintain consistency in our API hooks... the alternative approaches for when there are no toasts are in a few places... for example

- the `useShowConnectionsApi` test located around line `50` of this file... 
- the `useUserApi` test located in the `useLoginApi.test.ts` around line `160`...

Reply:
pushed fixup!

Reply:
added a minor clean up commit